test/pkg/test/framework.go

```diff
@@ -313,6 +318,36 @@ func (env *Framework) GetTargets(ctx context.Context, targetGroup *vpclattice.Ta
 	return found
 }
 
+func (env *Framework) ExpectEventuallyVpcOfCurrentClusterAssociateWithServiceNetwork(ctx context.Context, sn *vpclattice.ServiceNetworkSummary) {
+	currentClusterVpcId := os.Getenv("CLUSTER_VPC_ID")
+	fmt.Println("ExpectEventuallyVpcOfCurrentClusterAssociateWithServiceNetwork currentClusterVpcId:", currentClusterVpcId)
+	Eventually(func(g Gomega) {
+		vpcAssociations, err := env.LatticeClient.ListServiceNetworkVpcAssociationsAsList(ctx, &vpclattice.ListServiceNetworkVpcAssociationsInput{
+			ServiceNetworkIdentifier: sn.Id,
+			VpcIdentifier:            &currentClusterVpcId,
+		})
+		g.Expect(err).To(BeNil())
+		g.Expect(vpcAssociations).To(HaveLen(1))
+		for _, association := range vpcAssociations {
+			g.Expect(*association.Status).To(Equal(vpclattice.ServiceNetworkVpcAssociationStatusActive))
+		}
+	}).Should(Succeed())
+
+}
+
+func (env *Framework) ExpectEventuallyAllLatticeTargetsHealthy(ctx context.Context, tg *vpclattice.TargetGroupSummary) {
+	fmt.Printf("ExpectEventuallyAllLatticeTargetsHealthy for targetGroup: %+v\n", tg)
+	Eventually(func(g Gomega) {
+		targets, err := env.LatticeClient.ListTargetsAsList(ctx, &vpclattice.ListTargetsInput{TargetGroupIdentifier: tg.Id})
+		g.Expect(err).To(BeNil())
+		g.Expect(targets).ToNot(BeEmpty())
+		for _, target := range targets {
+			log.Println("target:", target)
+			g.Expect(*target.Status).To(Equal(vpclattice.TargetStatusHealthy))
+		}
+	}).WithPolling(time.Second * 5).WithTimeout(time.Minute * 3).Should(Succeed())
```

Comment:
I dont think pulling every 5 seconds is required here, can be relaxed to 10-15

Reply:
Not a fan of relying on default polling rates, API's are different. Changes in default can cause large blast radius. Please be explicit here